README.md

```diff
@@ -2,29 +2,36 @@
 
 Hybrid Application Console Infrastructure repository
 
-## Local Development
+## Run with webpack proxy
 
-Since the plugin is not served directly from console.redhat.com you have to run webpack proxy in order to properly develop locally. You have two options
+Since the plugin is not served directly from console.redhat.com you have to run webpack proxy in order to properly develop locally.
 
-* With hac-core served from CDN - if you want to test your changes just in your plugin
-* With hac-core locally - if you want to add new plugin or change how navigation is treated.
+### Update `/etc/hosts`
```

Comment:
You'll need to delete this section: https://github.com/openshift/hac-infra/blob/ba4c279d71826216cac5edda8c0524e6530019fb/README.md#proxies--etchosts